protos/escrow/escrow.proto

```diff
@@ -0,0 +1,101 @@
+syntax = "proto3";
+import "google/protobuf/timestamp.proto";
+
+package btfs;
+
+service EscrowService{
+	rpc SubmitContracts(EscrowContractRequest) returns (EscrowContractResult) {}
+	rpc PayIn(PayinRquest) returns (PayinResult) {}
+	rpc CheckPayment(ContractID) returns (PayinStatus) {}
+	rpc CancelContracts(CancelContractRequest) returns (CancelContractResult) {}
+};
+
+enum TokenType {
+	BTT = 0;
+}
+
+enum Schedule {
+	MONTHLY = 0;
+	QUARTERLY = 1;
+	ANNUALLY = 2;
+}
+
+message ContractID {
+	bytes contract_id = 1;
```

Comment:
suggest add signature here, as we maybe did not provide such check to public, maybe only renter, host and guard can be allowed to do such check.

Reply:
I was thinking open to public, by adding signature we can record who is doing the query. OK.